src/clj/fluree/db/query/sparql/translator.cljc

```diff
@@ -484,6 +542,5 @@
 
 (defn translate
   [parsed]
-  (reduce (fn [fql rule] (into fql (parse-rule rule)))
-          {}
-          parsed))
+  (->> parsed
+       (reduce (fn [fql rule] (into fql (parse-rule rule))) {})))
```

Comment:
this would be much more readable if you moved the function body after the argument vector to the next line